src/Command.ino

```diff
@@ -465,19 +467,84 @@ void ExecuteCommand(byte source, const char *Line)
     {
       success = true;
       if (Par2)
+      {
         //start new timer
-        RulesTimer[Par1 - 1] = millis() + (1000 * Par2);
+        RulesTimer[Par1 - 1].interval = Par2*1000;
+        RulesTimer[Par1 - 1].paused = false;
+        RulesTimer[Par1 - 1].timestamp = millis() + (1000 * Par2);
+      }
       else
+      {
         //disable existing timer
-        RulesTimer[Par1 - 1] = 0L;
+        RulesTimer[Par1 - 1].interval = 0;
+        RulesTimer[Par1 - 1].paused = false;
+        RulesTimer[Par1 - 1].timestamp = 0L;
+      }
     }
     else
     {
       addLog(LOG_LEVEL_ERROR, F("TIMER: invalid timer number"));
     }
     break;
   }
-
+  
+  case cmd_TimerPause:
+  {
+    if (Par1>=1 && Par1<=RULES_TIMER_MAX)
+    {
+       success = true;
+       if (RulesTimer[Par1 - 1].paused == false)
+       {
+          long delta = timeDiff(RulesTimer[Par1 - 1].timestamp, millis());
```

Comment:
Better use `long delta = timePassedSince(RulesTimer[Par1 - 1].timestamp);`
This function does exactly the same, but then it is more clear what it means and what negative values mean.

Reply:
Thanks, I fixed it.